packages/flutter_tools/lib/src/migrations/swift_package_manager_integration_migration.dart

```diff
@@ -64,18 +64,20 @@ class SwiftPackageManagerIntegrationMigration extends ProjectMigrator {
   /// Existing macOS identifier for Runner PBXFrameworksBuildPhase.
   static const String _macosRunnerFrameworksBuildPhaseIdentifier = '33CC10EA2044A3C60003C045';
 
-  /// Existing iOS identifier for Runner PBXNativeTarget.
```

Comment:
The PBXNativeTarget's identifier can be different if there are multiple targets which each have the application product type. I.e. the default Runner scheme is duplicated to quickly get going for a second application.